.github/workflows/main.yml

```diff
@@ -106,7 +112,7 @@ jobs:
         with:
           fetch-depth: 300
       - name: Fetch tags
-        run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+        run: git fetch --depth=300 origin +refs/tags/*:refs/tags/*
```

Comment:
The latest docs surprise me a bit though.

> 1.4.0+123.gaf86d38

https://github.com/streamlink/streamlink/commit/af86d38
https://github.com/streamlink/streamlink/runs/616381381?check_suite_focus=true#step:3:1

Why does it use `1.3.1` as the base (`+123`), but still refers to `1.4.0` in its output string? It should be `1.4.0+1`.
By the way, `1.4.0` is correct here, because the `1.4.1` tag wasn't pushed to the repo yet on the `1.4.1` release commit when the latest docs were built.

Is this a bug in versioneer? Does `git describe --tags` work?